Tools/sitl_run.sh

```diff
@@ -76,8 +76,21 @@ elif [ "$program" == "gazebo" ] && [ ! -n "$no_sim" ]; then
 		if  [[ -z "$DONT_RUN" ]]; then
 			# Set the plugin path so Gazebo finds our model and sim
 			source "$src_path/Tools/setup_gazebo.bash" "${src_path}" "${build_path}"
+			if [ -z $PX4_SITL_WORLD ]; then
+				#Spawn predefined world
+				if [ -z ${src_path}/Tools/sitl_gazebo/worlds/${model}.world ]; then
+					gzserver --verbose "${src_path}/Tools/sitl_gazebo/worlds/${model}.world" &
+				else
+					#Spawn empty world if world with model name doesn't exist
+					gzserver --verbose "${src_path}/Tools/sitl_gazebo/worlds/empty.world" &
+				fi
+			else
+				# Spawn world from environment variable
+				gzserver --verbose $PX4_SITL_WORLD &
```

Comment:
I would make this more intelligent, so that `PX4_SITL_WORLD` can be:
- an absolute path to a world file
- the name of the world, like `empty`

Reply:
@bkueng Since I added a make target for the worlds, would this be still necessary? `PX4_SITL_WORLD` should be only needed if the world is outside the `sitl_gazebo` repository now

Reply:
Done!